src/scripts/navigation-bar/index.js

```diff
@@ -30,11 +30,17 @@ export function NavigationBar() {
   )
   deleteNodesBtn.addEventListener("click", handleDeleteNodes, false)
   renameBtn.addEventListener("click", handleEditNode, false)
+  document.addEventListener("keyup", e => {
```

Comment:
I think that using the ` document` as the listened to element, like you did here, might be a bit overkill and is outside of the scope of the entire document